lib/ansible/modules/cloud/amazon/rds_instance.py

```diff
@@ -847,7 +847,7 @@ def get_options_with_changing_values(client, module, parameters):
     if cloudwatch_logs_enabled:
         parameters['CloudwatchLogsExportConfiguration'] = cloudwatch_logs_enabled
     if not module.params['storage_type']:
-        parameters.pop('Iops')
+        parameters.pop('Iops', None)
```

Comment:
I'm wondering if you could separate this so it can be backported.

Reply:
yeah, of course

Reply:
I've taken it out of this PR, once my currently running rds test suite is passing (I'm pretty confident it's all good now) I'll cherry pick onto a new branch

Reply:
Thanks so much